resources/easyrpg-player.6.adoc

```diff
@@ -28,7 +28,8 @@ for some additional binary patches.
   Disable audio (in case you prefer your own music).
 
 *--disable-rtp*::
-  Disable support for the Runtime Package (RTP).
+  Disable support for the Runtime Package (RTP). Will lead to checkerboard
+  graphics and silent music in games depending on the RTP.
```

Comment:
A bit nitpicky, but also silent sound effects. LGTM as is, too,

Reply:
Sure, can add this.